homeassistant/components/history_stats/config_flow.py

```diff
@@ -66,6 +66,17 @@ async def validate_options(
 )
 DATA_SCHEMA_OPTIONS = vol.Schema(
     {
+        vol.Optional(CONF_ENTITY_ID, description={"readonly": True}): EntitySelector(),
```

Comment:
See https://github.com/home-assistant/frontend/pull/22592#discussion_r1855173099

Reply:
renamed to `read_only`